content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+---
+title: Platform Customization
+section: root
+index: 45
+meta_description: "How Enterprise customers can create a custom environment and benefit from Active Traffic Management and other advanced Ably features."
+meta_keywords: "Ably, clusters, isolation, custom, CNAME, active traffic management"
+jump_to:
+  Contents:
+    - Active traffic management
+    - Dedicated and isolated clusters
+    - Regional routing and storage
+    - Setting up a custom environment
+---
+
+If you're an "Ably Enterprise":https://ably.com/pricing customer, then you can optionally create a *custom environment* to tailor the Ably platform to your specific requirements. A custom environment enables you to benefit from the following:
+
+- *Active traffic management*: Let Ably actively monitor not only the global cluster health but also your own endpoints and proactively take steps to isolate or move traffic to ensure business continuity.
+- *Dedicated, isolated clusters*: Rely upon guaranteed capacity and isolation from noisy neighbors.
+- *Regional routing of traffic*: Require that all traffic is routed and serviced within specific regions.
+- *Regional message storage*: Require that all messages are stored within a specific region.
```

Comment:
Will there be a deeper documentation page about this, or at least internal docs for CSMs? Because with Cassandra there are some serious downsides to forcing storage in only one region. (I know we have a few customers doing this – hopefully they were carefully informed of the trade-offs.)

Reply:
I have created a ticket for this to be considered: https://ably.atlassian.net/browse/DOC-572